### ISSUE TITLE
the  evaluation about the got-10k

### ISSUE THREAD

**Issue by meimeixu520** [ohhhyeahhh/SiamCAR#13]
Hi,thanks for your excellent work!  When I test the got-10k using the given model, then the results aren't same as the online test format requirements .So  I want  to ask how do you test it ? which format do you use?By the way, I didn't change anything.
Best regards!

**Comment by twotwo2** [comment]
We use got-10k toolkit to test the got-10k model. You can refer https://github.com/got-10k/toolkit

**Comment by meimeixu520** [comment]
Yes ,I know  ,but the result's format is not true.

**Comment by twotwo2** [comment]
It should be in a compressed format, such as a zip file.

**Comment by meimeixu520** [comment]
Yes, I know it shouble be zip file, what I mean by that is the final result of the code is not in the right format, for example:
![image](https://user-images.githubusercontent.com/37978749/86305109-75003500-bc43-11ea-9c78-458ca8f1d5bb.png)
but it need the following format:
![image](https://user-images.githubusercontent.com/37978749/86305138-88ab9b80-bc43-11ea-8b5a-732c3588647d.png)
![image](https://user-images.githubusercontent.com/37978749/86305155-919c6d00-bc43-11ea-987b-4e00e41687ba.png)

**Comment by twotwo2** [comment]
We will solve this problem later, thank you for your advice.

**Comment by meimeixu520** [comment]
Thanks for your reply! I look forward for your updating!
Best regards!
